src/pages/developing/web-components-tutorial/step-4.mdx

```diff
@@ -0,0 +1,365 @@
+  frameborder="no"
+  allowtransparency="true"
+  allowfullscreen="true"
+/>
+
+## Fork, clone and branch
+
+This tutorial has an accompanying GitHub repository called
+[carbon-tutorial-web-components](https://github.com/carbon-design-system/carbon-tutorial-web-components)
+that we'll use as a starting point for each step. If you haven't forked and
+cloned that repository yet, and haven't added the upstream remote, go ahead and
+do so by following the [step 1
+instructions]([previous step](/developing/web-components-tutorial/step-1#fork-clone-and-branch).
+
+### Branch
+
+With your repository all set up, let's check out the branch for this tutorial
+step's starting point.
+
+```bash
+git fetch upstream
+git checkout -b step-4 upstream/step-4
+```
+
+<InlineNotification>
+
+**Note:** This builds on top of step 3, but be sure to check out the upstream
+step 4 branch because it includes the static assets required to get through this
+step.
+
+</InlineNotification>
+
+### Build and start app
+
+Install the app's dependencies (in case you're starting fresh in your current
+directory and not continuing from the previous step):
+
+```bash
+pnpm i
+```
+
+Then, start the app:
+
+```bash
+pnpm dev
+```
+
+You should see something similar to where the
+[previous step](/developing/web-components-tutorial/step-1#fork-clone-and-branch/step-3)
+left off.
+
+## Review design
+
+Here's what we're building – an informational section that has a heading and
+three subheadings. Each subheading has accompanying copy and a pictogram. We'll
+assume that this informational section is used elsewhere on the site, meaning
+it's a great opportunity to build it as a reusable component. As for naming,
+we'll call it an `InfoSection` with three `InfoCard`s as children.
+
+![Info section layout](../shared/step-4/images/info-layout.png)
+
+<Caption>Info section layout</Caption>
+
+## The info section
+
+Inside `page-landing__r3` in `index.html` we will make a couple of small
+changes. Add the class `info-section` to the `cds--subgrid`
+
+```html path=index.html
+<div class="info-section cds--subgrid cds--subgrid--full-wide"></div>
+```
+
+and replace `The principles` and surrounding column with which changes the
+column settings and provides a class for us to style the section heading.
+
+```html path=index.html
+<div
+  class="info-section__column cds--sm:col-span-4 cds--md:col-span-8 cds--lg:col-span-16 cds--xlg:col-span-4 cds--css-grid-column">
+  <h3 class="info-section__heading">The Principles</h3>
+</div>
+```
+
+Style the `info-section__heading` with:
+
+```scss path=style.scss
+.info-section__heading {
+  @include type-style('heading-01');
+
+  padding-block-end: $spacing-08;
+}
+```
+
+## The info card
+
+At the bottom of `index.html` between the closing body and html tags add the
+info card template.
+
+### The template
+
+```html path=index.html
+<template id="template--info-card">
```

Comment:
this template too is also adding an extra `</template>` when copying and pasting

Reply:
Will investigate.

Reply:
Seems to have stopped doing it for me @ariellalgilmore I have changed nothing to affect this, other than merge a few suggested commits.